angularjs-portal-home/src/main/webapp/my-app/layout/partials/welcome.html

```diff
@@ -0,0 +1,8 @@
+<div class="welcome-popup center">
+  <h1>Welcome to the new MyUW. Here are the basics.</h1>
+  <img src="img/myuw-welcome.png">
```

Comment:
:+1: also. What do we think about this below the image? I like @timlevett's idea of an sr-only div. @keirserrie and @jiayinjx, thoughts on wording and whether this is helpful for screen reader users?

```
<div class="sr-only">
    <p>The new MyUW makes it easier to access your content. You can switch between simple icons and more informative widgets using 'Icon View' and 'Widget View' toggles. Each of the icons on the home page links to your MyUW content, and widgets can display a dashboard of information on the home page. If you are having trouble finding what you're looking for, you can use the top search box to search through all of MyUW's content, and save it to your home page from there using the 'Add to Favorites' button.</p>
</div>
```